src/Presentation/Nop.Web.Framework/Infrastructure/NopMvcStartup.cs

```diff
@@ -43,6 +43,9 @@ public void Configure(IApplicationBuilder application)
             //use WebMarkupMin
             application.UseNopWebMarkupMin();
 
+            //Add the EndpointRoutingMiddleware
+            application.UseRouting();
```

Comment:
Can you moveUseRouting to a stand-alone Startup File? Or give it to a Provinder method.
Or provide a way to insert custom middleware between UseRouting and UseNopEndpoints.

Reply:
Sorry, didn't get that you sort startups by order property. I've added a new startup file and place it before authentication in order.